pkg/rtc/mediatracksubscriptions.go

```diff
@@ -241,9 +256,6 @@ func (t *MediaTrackSubscriptions) AddSubscriber(sub types.LocalParticipant, wr *
 	subTrack.SetNeedsNegotiation(!replacedTrack)
 	subTrack.SetRTPSender(sender)
 
-	sendParameters := sender.GetParameters()
```

Comment:
The extension here is an offer extension that might not be valid for the client, move it to Downtrack.Bind